geniza/corpus/templates/admin/corpus/document/merge.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'admin/base_site.html' %}
+{% block title %} Merge selected documents {% endblock %}
+
+{% block breadcrumbs %}
+    {% if not is_popup %}
+        <ul>
+            <li><a href="{% url 'admin:index' %}">Home</a></li>
+            <li><a href="{% url 'admin:app_list' app_label='corpus' %}">Corpus</a></li>
+            <li><a href="{% url 'admin:corpus_document_changelist'%}">Document</a></li>
+            <li>Merge selected documents</li>
+        </ul>
```

Comment:
these tags don't match what other django admin templates output, looks like we need a `div` with `a` tags

Reply:
Done in ffe557a 👍